website/content/docs/agent/options.mdx

```diff
@@ -555,6 +555,10 @@ definitions support being updated during a reload.
 
 #### Example Configuration File, with TLS
 
+See [TLS Configuration Reference](#tls-configuration-reference) for more information about Agent TLS settings.
+
+~> **Security Note:** the three verify options must be set to `true` to enable TLS.
```

Comment:
Thanks! Yes I think that text is more correct.

The `auto_config` and `auto_encrypt` docs already go into that level of detail. What do you think about keeping this explanation a bit shorter, something like:

> all three verify options should be set as `true` to enable secure mTLS communication, enabling both encryption and authentication.

I'm also wondering if maybe this example should move to the new section I created in this PR, so that the example is next to the config settings. What do you think?

Reply:
I think what I was trying to say was that "If you do not set `verify_{incoming, outgoing}` then TLS is not enabled, even when you specify the ca and certs. I think our new wording doesn't make that clear necessarily. I'll see if I can find a way to explain it better.

Reply:
> The `auto_encrypt` docs already go into that level of detail. What do you think about keeping this explanation a bit shorter, something like: ...

I think that's a fair call, keeping it concise is probably best.

> I'm also wondering if maybe this example should move to the new section I created in this PR, so that the example is next to the config settings. What do you think?

I think that totally makes sense. 

> I think our new wording doesn't make that clear necessarily. I'll see if I can find a way to explain it better.

🤔 Hmm, gotch'a... maybe something like?:

```suggestion
~> **Security Note:** all three verify options should be set as `true` to enable secure mTLS communication, enabling both encryption and authentication. Failing to set [`verify_incoming`](#verify_incoming) or [`verify_outgoing`](#verify_outgoing) can result in TLS not being enabled at all, even when specifying a [`ca_file`](#ca_file), [`cert_file`](#cert_file), and [`key_file`](#key_file).
```

Reply:
That sounds awesome, thank you! I'll make those updates.